app/Notifications/Dto/PushoverMessage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Notifications\Dto;
+
+use Illuminate\Support\Facades\Log;
+
+class PushoverMessage
+{
+    public function __construct(
+        public string $title,
+        public string $message,
+        public array $buttons = [],
+        public string $level = 'info',
+    ) {}
+
+    public function getLevelIcon(): string
+    {
+        return match ($this->level) {
```

Comment:
Is it possible to use the same icons that we use for the other notification providers?

Reply:
The rest looks good to me so it is almost ready to be merged.

Reply:
I'm working on it 👍

Reply:
```php
return match ($this->level) {
            'info' => 'ℹ️',
            'error' => '❌',
            'success' => '✅ ',
            'warning' => '⚠️',
        };
```

Reply:
I've updated them to those